### PR TITLE
fix: fixed the relative path exporter

### PR DESCRIPTION
**Pull request by mike-ionut-mihai-sandu-tao**
##Description:
Fixed the relative path exporter

##Ticket: https://oat-sa.atlassian.net/browse/AUT-450

Env:   http://authoring-ionut.playground.kitchen.it.taocloud.org:45841/

### CODE REVIEW

File: model/portableElement/export/ImsPciExporter.php [hunk 1]
```diff
@@ -178,9 +178,9 @@ private function replaceFile($dataString, $fileToReplace)
     private function getItemRelativePath($itemBasePath)
     {
         $returnValue = '';
-        $arrDir = explode(DIRECTORY_SEPARATOR, rtrim($itemBasePath, DIRECTORY_SEPARATOR));
-        for ($i = 0; $i < count($arrDir); $i++) {
-            $returnValue .= '..' . DIRECTORY_SEPARATOR;
+        $arrDir = explode('/', rtrim($itemBasePath, '/'));
```
Comment (mike-ionut-mihai-sandu-tao) [inline model/portableElement/export/ImsPciExporter.php:181]: provide a descriptive constant  instead of '/'

Comment (mike-ionut-mihai-sandu-tao) [reply model/portableElement/export/ImsPciExporter.php]: replaced by EXPORTER_PATH_SEPARATOR

File: model/portableElement/export/ImsPciExporter.php [hunk 2]
```diff
@@ -21,15 +21,16 @@
...
-use oat\taoQtiItem\model\portableElement\export\PortableElementExporter;
 use oat\taoQtiItem\model\portableElement\element\PortableElementObject;
-use \DOMDocument;
-use \DOMXPath;
+use oat\taoQtiItem\model\portableElement\export\PortableElementExporter;
+use oat\taoQtiItem\model\portableElement\PortableElementService;
 
 class ImsPciExporter extends PortableElementExporter
 {
+    const EXPORTER_PATH_SEPARATOR = '/';
```
Comment (shpran) [inline model/portableElement/export/ImsPciExporter.php:33]:
```suggestion
    private const EXPORTER_PATH_SEPARATOR = '/';
```

File: model/portableElement/export/ImsPciExporter.php [hunk 3]
```diff
@@ -178,9 +179,9 @@ private function replaceFile($dataString, $fileToReplace)
     private function getItemRelativePath($itemBasePath)
```
Comment (shpran) [inline model/portableElement/export/ImsPciExporter.php:179]:
```suggestion
    private function getItemRelativePath($itemBasePath): string
```

File: model/portableElement/export/ImsPciExporter.php (hunk 2 above)
Comment (mike-ionut-mihai-sandu-tao) [reply model/portableElement/export/ImsPciExporter.php]: fixed

File: model/portableElement/export/ImsPciExporter.php (hunk 3 above)
Comment (mike-ionut-mihai-sandu-tao) [reply model/portableElement/export/ImsPciExporter.php]: fixed

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**